### ISSUE TITLE
MSVC_VERSION is incorrect when cmake configure project with tool kit clang-cl 

### ISSUE THREAD

**Issue by L-Sun** [microsoft/vscode-cmake-tools#1053]
### Brief Issue Summary
When I configure my project, the cmake variable `${MSVC_VERSION}` output incorrectly.
Here is my CMakeLists.txt
```cmake
cmake_minimum_required(VERSION 3.15)

project(MyProject)

add_executable(main main.cpp)
if(MSVC)
    message("MSVC_VERSION: ${MSVC_VERSION}")
endif(MSVC)
```

### Expected:

1. run command CMake: Delete Cache and Reconfigure
2. and Output panel will show:
```
...
[cmake] MSVC_VERSION: 1924
[cmake]  
[cmake] -- Configuring done
[cmake] -- Generating done
```

### Apparent Behavior:

1. run command CMake: Delete Cache and Reconfigure
2. and Output panel show the following message
```
...
[cmake] MSVC_VERSION: 1911
[cmake]  
[cmake] -- Configuring done
[cmake] -- Generating done
```

### Other Notes/Information

When I add `'VCToolsInstallDir'` to `MSVC_ENVIRONMENT_VARIABLES` array in `\.vscode\extensions\ms-vscode.cmake-tools-1.3.0\dist\main.js`(or src/kit.ts) of extension.
and reconfigure my project. It works as expected. The cmake variable `${MSVC_VERSION}` is correct.

### Platform and Versions

- **Operating System**: Windows
- **CMake Version**: 3.16.4
- **VSCode Version**: 1.42.0
- **CMake Tools Extension Version**: 1.3.0
- **Compiler/Toolchain**:  clang-cl (11.0.0) + VS2019 (v16.4.4)
- **visualStudioArchitecture**: amd64

**Comment by tklajnscek** [comment]
+1 
Just hit the same issue where Clang can't find the MSVC toolkit correctly. 
Traced it down to this after many hours.

Is there any specific reason the `VCToolsInstallDir` is omitted from `MSVC_ENVIRONMENT_VARIABLES `? Seems like someone generated this a while ago and it never got updated.

Clang uses `VCToolsInstallDir` to find the location of the toolkit.

See `https://clang.llvm.org/doxygen/MSVC_8cpp_source.html` -> `findVCToolChainViaEnvironment()`

**Comment by tklajnscek** [comment]
Oh, just to be clear - in my case this doesn't simply mess up the MSVC_VERSION. 
It completely fails to compile anything because the include/lib paths are set to non-existing locations.

**Comment by andreeis** [comment]
This fix is included in the 1.9.0 CMake Tools release. Please upgrade your extension in VSCode and let us know if you encounter any other issues.
